amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -837,6 +840,77 @@ type IngestorStats @model {
   value: String!
 }
 
+type SimpleTargetProfile {
+  avatarHash: String
+  displayName: String
+  username: String
+}
+
+type SimpleTarget {
+  id: String!
+  profile: SimpleTargetProfile!
+  walletAddress: String!
+}
+
+type NFT {
+  id: String!
+  profile: NFTProfile!
+  walletAddress: String!
+}
+
+type NFTProfile {
+  displayName: String!
+}
+
+type SafeBalanceToken {
+  name: String!
+  decimals: Int!
+  symbol: String!
+  address: String!
+  networkName: String
+  logoUri: String
+}
```

Comment:
also I believe in a previous PR you mentioned that we are not using `networkName` anymore so is this ok to remove?

Reply:
> why can't we re use the `Token` type here?

I think we can. Do you think it would be worth it to create a new `Token` entry in the DB if the safe balance's token is not there yet? We have been using this type only for colony tokens until now that is.

Reply:
I do think it's worth creating a new `Token` entry if the safe balance's token is not there yet. We need to store this somewhere after all :)

Though I didn't realise we are only using this type for colony tokens this enum in the `schema.graphql` confused me:
```
enum TokenType {
  COLONY
  ERC20
  CHAIN_NATIVE
}
```
could we add make the safe tokens an `ERC20` type?

Reply:
> We need to store this somewhere after all :)

Just to clarify, the tokens used in the safe transactions are already stored in the `SafeTransaction` model, and that data is only used on the action/motion page.

> could we add make the safe tokens an `ERC20` type?

I think so, although we don't have a utility for that. At least not yet.

I'll create a separate issue to further explore this as these tokens used in safe transactions are presumably tokens from a foreign chain and right now the query that we use to add new tokens to the DB ("fetchTokenFromChain" in `amplify`) assumes that they are in the same chain. So I expect some significant changes to be made to adapt it if we want to go this route.